src/test/acceptance/features/address-lookup.feature

```diff
@@ -3,22 +3,36 @@ Feature: Select address
   As a potential claimant
   I want to lookup my address
 
-  Background:
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario: Entering a postcode with no search results shows that no addresses were found
     Given I have entered my details up to the postcode page
+    When I enter a postcode with no search results
```

Comment:
Maybe `When I enter a postcode that returns no search results`?

Reply:
Updated